examples/candle.py

```diff
@@ -21,10 +21,10 @@
     wait = np.random.choice(np.random.noncentral_chisquare(5, 1, 1000), 1)[0] / 50
     n = np.random.choice(np.random.noncentral_chisquare(5, 0.1, 1000), 1)
     limit = int(n[0])
-    if limit > 8:
-        limit = 8
+    if limit > NUM_PIXELS:
+        limit = NUM_PIXELS
     for pixel in range(limit):
-        hue = start + (((end - start) / 8.0) * pixel) 
+        hue = start + (((end - start) / (NUM_PIXELS*1.0)) * pixel) 
```

Comment:
I think `float(NUM_PIXELS)` would be preferable to `(NUM_PIXELS*1.0)` here (technically correct, I guess?) since it's an implicit coercion to float, rather than relying on the implicit side-effect of multiplying an integer by a float.